src/pybind/mgr/rook/module.py

```diff
@@ -431,6 +433,9 @@ def remove_service(self, service_name: str) -> str:
             return self.rook_cluster.rm_service('cephobjectstores', service_name)
         elif service_type == 'nfs':
```

Comment:
Ceph auth keys for the nfs service are not deleted. This is an operation to do in **ALL** the orchestrators

```
# ceph auth ls

client.nfs-ganesha.jmonfs.a
	key: AQChg0Bh5V7ZHRAAFMydhUKjSPKvZRxV7qTBsg==
	caps: [mon] allow r
	caps: [osd] allow rw pool=.nfs namespace=jmonfs

```